tsdb/index/tsi1/index.go

```diff
@@ -27,7 +27,7 @@ const IndexName = "tsi1"
 // Default compaction thresholds.
 const (
 	DefaultMaxLogFileSize   = 5 * 1024 * 1024
-	DefaultCompactionFactor = 1.8
+	DefaultCompactionFactor = 10 // 1.8
```

Comment:
Can you add a comment to this constant?  The value and commented out `// 1.8` are very cryptic.